.github/workflows/release.yml

```diff
@@ -0,0 +1,64 @@
+name: Quarkiverse Release
```

Comment:
This is outdated, can you make sure you have the latest Quarkus CLI installed?

Reply:
you are right, it's outdated 😓 , let me iterate again